API.md

```diff
@@ -0,0 +1,136 @@
+# Mail Ballot Manager APIs
+
+## Open Questions
+
+- If there is an error in the election data, how does a election official start
+  over?
+- If there is an error in the voter mailing list, how does an election official
+  fix it?
+- If an election official needs to stop the presses, can they?
+
+## Types
+
+Election as defined in `@votingworks/ballot-encoder` and the following…
+
+```
+interface VoterMailBallot {
```

Comment:
Are any of these fields intended to be optional?

Reply:
Yes. Optionality should be indicated.

Reply:
https://github.com/votingworks/mail-ballot-manager/pull/15